src/wazuh_modules/vulnerability_detector/wm_vuln_detector_db.h

```diff
@@ -53,6 +53,8 @@ typedef enum vu_query {
     VU_REMOVE_UNUSED_ID,
     VU_JOIN_QUERY,
     VU_JOIN_RH_QUERY,
+    VU_JOIN_DEBIAN_QUERY,
```

Comment:
In this enumeration, one of these four is left over:

```
VU_JOIN_QUERY,
VU_JOIN_RH_QUERY,
VU_JOIN_DEBIAN_QUERY,
VU_GET_VULN_INFO,
```

according to the length of the `vu_queries[]` array.

Reply:
Forget it, I didn't see the last query.